docs/modules/ROOT/pages/llms.adoc

```diff
@@ -22,5 +22,4 @@ Note that each LLM has a different feature set.
 Please check the specific documentation for the LLM you are using to see what features are available:
 
 - xref:openai.adoc[OpenAI (GT-3/4)]
-- xref:hugging-face.adoc[Hugging Face]
-- xref:ollama.adoc[Ollama]
\ No newline at end of file
+- xref:huggingface.adoc[Hugging Face]
```

Comment:
Ah yeah, I had let these ones because I thought Clément wanted to add them soon.